src/mlagility/cli/cli.py

```diff
@@ -225,10 +226,17 @@ def check_extension(choices, file_name):
         action="store_true",
     )
 
+    benchmark_parser.add_argument(
+        "--export-only",
+        dest="export_only",
+        help="Stop this command after the ONNX export in the build phase",
```

Comment:
nit: This comment assumes all `benchit` calls are for benchmakring and this flag cuts that in short. Is that intended?